x-pack/plugins/reporting/export_types/common/execute_job/get_absolute_url.test.ts

```diff
@@ -7,24 +7,26 @@
 import { memoize } from 'lodash';
 import { getAbsoluteUrlFactory } from './get_absolute_url';
 
-const createMockServer = ({ settings = {} } = {}) => {
+const createMockServer = ({ settings = {} }: any) => {
```

Comment:
What about extracting this out into a test helper function so it can be reused instead of redefining the mock server in all the tests?

Reply:
hmm, maybe `x-pack/plugins/reporting/export_types/common/execute_job/test_helpers/create_mock_server.js` ?

Reply:
or could move it higher, maybe `x-pack/plugins/reporting/test_helpers/create_mock_server.js`

Reply:
Yea I like it there better. this way if any of the tests in reporting can use it and its not just for export types